packages/hardhat/contracts/strategy/CrabStrategy.sol

```diff
@@ -190,10 +199,9 @@ contract CrabStrategy is StrategyBase, StrategyFlashSwap, ReentrancyGuard, Ownab
      * @param _capAmount the maximum strategy collateral in ETH, checked on deposits
      */
     function setStrategyCap(uint256 _capAmount) external onlyOwner {
-        uint256 oldCap = strategyCap;
```

Comment:
saving bytecode by not emiting the old values for all these setters

Reply:
Or can emit event before changing value, but this works also!

Reply:
Any preference from your side?